Playbooks/RecordedFuture-Block-IPs-and-Domains-on-Microsoft-Defender-for-Endpoint/RecordedFuture-TIforDefenderEndpoint.json

```diff
@@ -2,82 +2,117 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "metadata": {
-		"comment": "This playbook leverages Recorded Future API and automatically imports the Command and Control IPs and Weaponized Domains Security Control Feeds, as Threat Intelligence Indicators, for prevention purposes in Microsoft Defender for Endpoint",
-		"author": "Glenn Wong & Adrian Porcescu, Recorded Future"
-	},
-	"parameters": {
+        "title": "RecordedFuture-TIforDefenderEndpoint",
+        "description": "This playbook leverages Recorded Future API and automatically imports the Command and Control IPs and Weaponized Domains Security Control Feeds, as Threat Intelligence Indicators, for prevention purposes in Microsoft Defender for Endpoint",
+        "prerequisites": [
+            "First install the RecordedFuture-TIforDefenderEndpoint playbook.",
+            "To use the Recorded Future for Azure connector, you will need a valid API token from Recorded Future as described in the documentation https://learn.microsoft.com/en-us/connectors/recordedfuturev2/#how-to-get-credentials"
+        ],
+        "postDeployment": [
+            "After deployment you have to open the playbook to configure all connections and press save."
+        ],
+        "prerequisitesDeployTemplateFile": "",
```

Comment:
This playbook has dependency on RecordedFuture-ImportToDefenderEndpoint.json. So please add prerequisitesDeployTemplateFile value. Since they are in same folder it will be ./RecordedFuture-ImportToDefenderEndpoint.json. Like,
"prerequisitesDeployTemplateFile": "./RecordedFuture-ImportToDefenderEndpoint.json"